webplugin/js/app/zendesk-chat-service.js

```diff
@@ -85,4 +106,35 @@ function ZendeskChatService() {
             },
         });
     };
+
+    _this.handleZendeskAgentFileSendEvent= async function (event) {
+        // window.console.log("handleZendeskAgentFileSendEvent ",event);
+
+        // let {url, mime_type, name, size}=event.attachment; 
+        var messagePxy = {
```

Comment:
We can remove async as we are not using await and comments if not required

Reply:
done